plugins/kubernetes/app/models/kubernetes/release_doc.rb

```diff
@@ -56,9 +56,10 @@ def update_replica_count(new_count)
 
       if replicas_live >= replica_target
         self.status ='live'
-      elsif replicas_live > 0
+      elsif replicas_live > 0 && !failed?
```

Comment:
Hummm, good point. With the guard at the beginning of the method we would never be able to recover if resources become available.

I believe that's fine for now, but eventually, what we could do as well, is mark that Pod as 'Failed' when we get an error, in the internal data structure, and remove that "marker" if we get a new event for that Pod with the condition `Ready` set to `True`, in which case we could also update the status of the ReleaseDoc back to 'spinning-up' or 'live', depending on the new number of live replicas. The advantage of doing this is a more accurate update of the UI.

Food for thoughts when we come back to this?